extensions/wikia/AssetsManager/config.php

```diff
@@ -293,7 +293,10 @@
 		// IL recovery
 		'//extensions/wikia/ARecoveryEngine/js/InstartLogic/recovery.js',
 
-		'//extensions/wikia/AdEngine/js/wrappers/prebid.js'
+		'//extensions/wikia/AdEngine/js/wrappers/prebid.js',
+
+		// prebid library
+		'#group_pr3b1d_prod_js'
```

Comment:
Can we add comment to remove `pr3b1d_prod_js` group and move .js file to this group "after ADEN-6797 release"?

Reply:
Done: https://github.com/Wikia/app/pull/14965/commits/a20d0e1b3f3836bc3cc111a2b46199ab3b96b480